app/containers/TopicHighlighter/index.js

```diff
@@ -42,6 +42,25 @@ export class TopicHighlighter extends Component {
     }
   }
 
+  articleNotLoaded() {
+    if(this.props.article.isFetching) {
+      return (
```

Comment:
Let's pair these attributes to a CSS selector and toggle that on/off based on `isFetching`